Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Dockerfile for cBioPortal
+FROM elementolab/jamm:1.0
```

Comment:
Agree 100% in adopting an official image and any improvements to reduce size.

To the best of my knowledge tomcat is lacking maven and mysql which are both required components during build.

Current [dockerfile] (https://hub.docker.com/r/elementolab/jamm/~/dockerfile/) builds from official centos and installs all cBioPortal prereqs from scratch.

We can switch to tomcat but we would still need to install all prereqs. Any suggestions?